### PR TITLE
replaced unknown with 'sans-serif'

### PR DESCRIPTION
**Pull request by awalGarg**
With reference to: https://bugzilla.mozilla.org/show_bug.cgi?id=1112740#c16

### CODE REVIEW

**Comment by wilsonpage** [comment]
@gmarty you happy with this?

**Comment by awalGarg** [comment]
@wilsonpage I have to add one more commit (adding a test is required in font_fit_test.js) to complete this patch, so I guess we can delay merging this for a while?

**Comment by gmarty** [comment]
I don't object this change because the font face should always be set.
And yeah a basic test would be nice. Thanks!

File: lib/font-fit.js [hunk 1]
```diff
@@ -227,7 +227,7 @@
       }
 
       return {
-        fontFamily: style.fontFamily || 'unknown',
+        fontFamily: heading.style.fontFamily || 'sans-serif',
```
Comment (julienw) [inline lib/font-fit.js]: why did you add "heading" here ?

**Comment by julienw** [comment]
@wilsonpage should this change be done in https://github.com/gaia-components/font-fit now ?

File: lib/font-fit.js (hunk 1 above)
Comment (awalGarg) [inline lib/font-fit.js]: `style.fontFamily` would always be truthy, since browser would assign "serif" as the default value, reflected in the return value of `getComputedStyle`.

Comment (julienw) [inline lib/font-fit.js]: but if you don't use the computed style, this will always be empty. I think the issue here is that there are cases where the style resolves to `''`.

Comment (awalGarg) [inline lib/font-fit.js]: I see. But how can we reproduce that case (in a test)?

**Comment by wilsonpage** [comment]
This is no longer valid as font-fit.is is not an external repo and we just pass the font shorthand string now, not individual font parameters.

**Comment by wilsonpage** [comment]
*now an external repo
